config/locales/en.yml

```diff
@@ -96,11 +124,10 @@ en:
     bibliographic_id: |
       Bibliographic ID should be used for an external record identifier that can connect the Avalon item to a catalog record or other record for the original item. Type specifies the type of external record identifier.
     language: |
-      Language describes the languages used in the item. Typing two or more
+      <strong>Required field.</strong>Language describes the languages used in the item. Typing two or more
```

Comment:
Missing space? I think this will render as *Required field.Language describes..." etc?

Reply:
Cheers, fixed this and elsewhere.